archive/terraform_critical/Makefile

```diff
@@ -0,0 +1,18 @@
+ROOT = $(shell git rev-parse --show-toplevel)
+include $(ROOT)/functions.Makefile
+
+STACK_ROOT = archive/terraform_critical
+
+SBT_APPS 	=
+SBT_DOCKER_LIBRARIES =
+SBT_NO_DOCKER_LIBRARIES =
+
+ECS_TASKS 	=
+LAMBDAS 	=
+
+TF_NAME 	= archive_critical
```

Comment:
nit: Could we swap the name to be `critical_archive`?

Reasoning: `make archive` could autocomplete to the critical or non-critical stacks; putting `critical` first ensures it can't be accidentally autocompleted.